index.html

```diff
@@ -30,7 +30,7 @@
         <li><a href="amsterdam.html">Amsterdam</a></li>
         <li><a href="rotterdam.html">Rotterdam</a></li>
         <li><a href="the_hague.html">The Hague</a></li>
-        <li><a href="utrecht.html">Utrecht</a></li>
+        <li><a href="utrecht.html">Utrecht 1</a></li>
```

Comment:
```suggestion
        <li><a href="utrecht.html">Utrecht</a></li>
```

Seem like you've made a typo!